README.md

```diff
@@ -15,45 +15,95 @@ Requires Rust 1.51.
 
 Create a new temporary project:
 
-* With no additional dependencies:
-    `$ cargo-temp`
-    
-* With multiple dependencies:
-    `$ cargo-temp rand tokio`
-    
-* When specifying a version:
-    `$ cargo-temp anyhow=1.0`
-    * Using the [cargo's comparison requirements][comparison]:
-        `$ cargo-temp anyhow==1.0.13`
+### With no additional dependencies:
+
+```bash
+cargo-temp
+```
+
+### With multiple dependencies:
+
+```bash
+cargo-temp rand tokio
+```
+
+### When specifying a version:
+
+```bash
+cargo-temp anyhow=1.0`
+```
+
+Using the [cargo's comparison requirements][comparison]:
+
+* Exact version:
+  ```bash
+  cargo-temp anyhow==1.0.13
+  ```
+
+* Maximal version:
+  ```bash
+  cargo-temp anyhow=<1.0.2
+  ```
 
 ### Repositories
 
-You can add repositories to your `Cargo.toml`.
+You can add repositories to your *Cargo.toml*.
 
 Examples:
 
-* HTTP
-    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git`
+* HTTP:
+  ```bash
+  cargo-temp anyhow=https://github.com/dtolnay/anyhow.git
+  ```
 
 * SSH
-    `$ cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git`
+  ```bash
+  cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git
+  ```
 
 To choose a branch or a revision:
 
-* Branch
-    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=master`
+* Branch:
+  ```bash
+  cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=master
+  ```
 
-* Revision
-    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#rev=7e0f77a38`
+* Revision:
+  ```bash
+  cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#rev=7e0f77a38
+  ```
 
-Without a branch or a revision, cargo will use the default branch of the repository.
+Without a branch or a revision, cargo will use the default branch of the
+repository.
 
 ## Features
 
+### The TO_DELETE file
+
 If you change your mind and decide to keep the project you can just delete the
 `TO_DELETE` file and the directory will not be deleted when the shell or the
 editor exits.
 
+### Git Working Tree
+
+You can create a git worktree from the current repository using:
+
+```bash
```

Comment:
Not bash.

If you add bash on command line examples it might add some confusing colors. I know it's technically bash but it's less readable for command lines.

(Also it's not really bash as those commands can be executed on other shells)

Reply:
Okey, [cargo-edit](https://raw.githubusercontent.com/killercup/cargo-edit/master/README.md) use `sh` but i already seen `console`.
What do you think about those ?

Reply:
Just use nothing. Nothing == text, no color. Do you need colors? No. Then don't use color syntax XD